project/Dependencies.scala

```diff
@@ -104,7 +104,7 @@ object Dependencies {
   val mysqlConnector: ModuleID =  "mysql"                         % "mysql-connector-java"  % "8.0.22"
   val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "3.10.3"
 
-  val workbenchLibsHash = "7ddf186"
+  val workbenchLibsHash = "11a45ad"
```

Comment:
where did `` come from? I see `d7ed6bf` as latest on https://github.com/broadinstitute/workbench-libs/blob/develop/util/CHANGELOG.md

Reply:
See the changelog here for the workbench-google2 lib! 
https://github.com/broadinstitute/workbench-libs/blob/develop/google2/CHANGELOG.md
And here from the main readme: 
https://github.com/broadinstitute/workbench-libs#workbench-google2